res/xml/persian.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key key0="ض" key2="١" key3="`" key4="esc"/>
+    <key key0="ص" key2="٢" key3="\@"/>
+    <key key0="ث" key2="٣" key3="\#"/>
+    <key key0="ق" key2="٤" key3="$"/>
+    <key key0="ف" key2="٥" key3="%"/>
+    <key key0="غ" key2="٦" key3="^"/>
+    <key key0="ع" key2="٧" key3="&amp;"/>
+    <key key0="ه" key2="٨" key3="*"/>
+    <key key0="خ" key2="٩" key3="("/>
+    <key key0="ح" key2="٠" key3=")"/>
+    <key key0="ج"/>
+  </row>
+  <row>
+    <key key0="ش" key4="tab"/>
+    <key key0="س"/>
+    <key key0="ی"/>
+    <key key0="ب"/>
+    <key key0="ل"/>
+    <key key0="ا" key1="آ" key2="!"/>
+    <key key0="ت" key1="&#x200C;"/>
```

Comment:
This is the right hand side of the keyboard. Perhaps `key2` is easier to reach ?

Reply:
In my opinion `key2` is suitable for it. And also for math operators I think this is enough. If not we can change it later!